metadata-ingestion/src/datahub/ingestion/source/looker/lookml_config.py

```diff
@@ -165,6 +165,15 @@ class LookMLSourceConfig(
         "view.sql_table_name. Defaults to an empty dictionary.",
     )
 
+    lookml_parameter: Dict[str, str] = Field(
+        {},
+        description=(
+            "A dictionary containing LookML parameters (`@{param_name}`) and their values. "
+            "Parameters are used to define dynamic references in LookML views, such as `view.sql_table_name`."
+            "Defaults to an empty dictionary."
```

Comment:
also update the liquid variable docs to clarify that they use the `${var}` syntax